ui/pages/confirmations/components/confirm/info/typed-sign/typed-sign-v4-simulation/decoded-simulation/decoded-simulation.tsx

```diff
@@ -60,18 +60,20 @@ const StateChangeRow = ({
   stateChangeList,
   stateChange,
   chainId,
+  displayLabel,
 }: {
   stateChangeList: DecodingDataStateChanges | null;
   stateChange: DecodingDataStateChange;
   chainId: Hex;
+  displayLabel: boolean;
```

Comment:
Minor, should this be `shouldDisplayLabel` to make it clear it's a boolean?

Reply:
updated